packages/venia-ui/lib/components/Header/container.js

```diff
@@ -1,17 +1,20 @@
 import { connect } from '@magento/venia-drivers';
 
-import Header from './header';
 import { toggleSearch } from '../../actions/app';
+import { getCartDetails, toggleCart } from '../../actions/cart';
```

Comment:
How much work would it be to write a new query/action that only requests the item count? I know the efficiency of our gql calls has been under the microscope lately so maybe a smaller query would help, especially for something that shows up on the header/initial load.

Reply:
I thought this was a great suggestion but unfortunately implementing it uncovered [a bug in our GraphQL](https://github.com/magento/graphql-ce/issues/830) where the item quantity is not being returned correctly.

Reply:
😢